### ISSUE TITLE
Running npm release fails - Unexpected token

### ISSUE THREAD

**Issue by jbbr** [kern/filepizza#13]
_System:_
Arch Linux
NodeJS v0.12.4
NPM 2.11.2

Installed with ```# npm install -g filepizza``` and started with ```$ filepizza```

_Error:_

```bash
$ filepizza      
/usr/lib/node_modules/filepizza/lib/server.js:39
    db.create(socket).then((u) => {
                               ^^
SyntaxError: Unexpected token =>
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Module._extensions..js (module.js:478:10)
    at Object.require.extensions.(anonymous function) [as .js] (/usr/lib/node_modules/filepizza/node_modules/babel/lib/babel/api/register/node.js:130:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/lib/node_modules/filepizza/index.js:3:18)
    at Module._compile (module.js:460:26)
```

Same problem when using iojs 2.x. Seems to be a problem with babel transpiling. iojs should support "() =>" functions out of the box but still same error.

**Comment by jbbr** [comment]
By the way: Latest git master runs without problems when using ```npm start```

**Comment by kern** [comment]
Thanks for pointing this out! Fix pushed in `0.1.1`.

Had to set up the Babel register hook properly since global installation causes all the files in `lib/` to be ignored.
